policies/coco-operating-policies.md

```diff
@@ -0,0 +1,120 @@
+decision.
+
+*Moderation*
+
+There is no dedicated chair.  Rather, the role of chair is taken on a
+per-meeting basis by a moderator who both prepares the agenda and moderates
+the discussion. The moderator role rotates through all CoCo members on a
+specified cycle (with swaps as needed to accommodate
+vacations/conflicts/etc).
+
+*Agenda/Notes*
+
+All CoCo members are able to add items to the agenda at any time, subject to
+approval of the meeting moderator. Notes are taken in the same document as
+the agenda. The CoCo is collectively responsible for the accuracy of the
+notes.
+
+## Voting
+
+The CoCo aims for consensus when making decisions.  However if this is
+impractical in a timely manner (i.e. the committee is deadlocked), or the
+item under discussion is judged to be important enough by assent of the
+committee, a vote may be requested. The following are additional useful
+guidelines for votes:
+
++ If a choice is change vs stay the same, status quo should be default - hence ties
+  should generally lead to status quo.
++ If the CoCo is deadlocked, the issue may be better put to the broader community or
+  the Astropy Voting Membership. This is particularly true for especially “weighty”
+  decisions, and the CoCo should feel free to put the question to the broader
+  community in place of decision by close votes.
+
+## Outward Communication Channels
+
++ The running notes should be open.
+
++ To make announcements to the Astropy community, the CoCo generally relies upon
+  relevant mailing lists and forums.  When sending such announcements from the CoCo,
+  a CoCo member should be careful to sign a message as:
+
+        For the Astropy Coordination Committee,
+        <Name>
+
++ When sending a message as an individual member of the project (or in one of their
+  other roles), they should sign in whatever other way they prefer.
+
+## Internal Communication channels
+
+*Coordinators Mailing List/Google group (coordinators@astropy.org)*
+
+Used for private conversations between the coordinators and specific individuals or
+just among the coordinators. There is an expectation that messages in this group might
+be private communication.  Hence, without the CoCo’s agreement, these emails are not
+to be shared outside of the CoCo, although future CoCos will be able to see them.
+
+*Private Slack channel (`#coordinatorsNN`)*
+
+Used for private, informal conversations among CoCo members. A new channel is created
+each time CoCo membership changes. There is a strong expectation of privacy and
+messages should not be shared without express permission of the sender.
+
+*Other Project Emails (e.g. confidential@astropy.org)*
+
+As with the coordinators list, there is generally an expectation of privacy from those
+who send to these emails, although that varies with the email.
+`confidential@astropy.org` has particular status in that it goes direct to the
+Project Ombudsperson.
```

Comment:
In fact, I would take that out her. confidential@ does not go to the Coco, so it should not be in the Coco operating procedures. We don't want people to think that the Coco can (or even would!) snoop on communication to the ombdbusperson.

Reply:
I've removed this